src/Worker/Grpc/InternalOptionsExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.DurableTask.Worker.Grpc.Internal;
+
+/// <summary>
+/// Provides access to configuring internal options for the gRPC worker.
+/// </summary>
+public static class InternalOptionsExtensions
```

Comment:
Please include disclaimers as seen on this type: https://github.com/microsoft/durabletask-dotnet/blob/main/src/Abstractions/Internal/IOrchestrationSubmitter.cs

Reply:
I have added an XML comment on the extension method